tests/e2e/e2e_test.go

```diff
@@ -825,5 +825,92 @@ func TestE2E(t *testing.T) {
 				return nil
 			},
 		},
+		testExecution{
+			Name: "TestPlatformAndNodeSuiteScan",
+			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, mcTctx *mcTestCtx, namespace string) error {
+				suiteName := "test-suite-two-scans-with-platform"
+
+				workerScanName := fmt.Sprintf("%s-workers-scan", suiteName)
+				selectWorkers := map[string]string{
+					"node-role.kubernetes.io/worker": "",
+				}
+
+				platformScanName := fmt.Sprintf("%s-platform-scan", suiteName)
+
+				exampleComplianceSuite := &compv1alpha1.ComplianceSuite{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      suiteName,
+						Namespace: namespace,
+					},
+					Spec: compv1alpha1.ComplianceSuiteSpec{
+						AutoApplyRemediations: false,
+						Scans: []compv1alpha1.ComplianceScanSpecWrapper{
+							{
+								ComplianceScanSpec: compv1alpha1.ComplianceScanSpec{
+									ContentImage: "quay.io/jhrozek/ocp4-openscap-content:ignition_remediation",
```

Comment:
You can already use ` ContentImage: "quay.io/complianceascode/ocp4:latest"` to minimise usage of personal repositories in the codebase.

Reply:
Follow-up